portfolio/src/main/webapp/background.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" content="width=device-width, initial-scale=1"> 
+    <title>Background</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body id="sub-page">
+    <div class="sidenav">
+      <a href="index.html">Home</a>
+      <a href="background.html">Background</a>
+      <a href="projects.html">Projects</a>
+      <a href="personal.html">Personal</a>
+      <a href=
+            "https://www.linkedin.com/in/theodora-mara-pislar-983117196/"
```

Comment:
Please use 4 space lin wrapping, also I think this url can fit on line above

Reply:
I agree that the url fits one line above, but about the line wrapping I checked the styleguide and it seems fine: "When line-wrapping, each continuation line should be indented at least 4 additional spaces from the original line." from here: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping